sql/sql.camel.yaml

```diff
@@ -0,0 +1,20 @@
+- route:
```

Comment:
Does it work with plain camel? what about quarkus export?

Reply:
You can add both as quarkus has a quarkus.xxx style. 

But the point is not so much about SB vs Q but to show Camel can do with SQL.